assignments/objects.js

```diff
@@ -49,16 +91,36 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: 'Susan',
+  age: '70',
+  parentspeak: () => `Hello, I'm the parent ${parent.name}!!`,
+  child: {
+    name: 'George',
+    age: 50,
+    childspeak: () => `Hello, I'm the child ${parent.child.name}!!`,
+    grandchild: {
+      name: 'Sam',
+      age: 30,
+      grandchildspeak: () => `Hello, I'm the grandchild ${parent.child.grandchild.name}!!`
+    }
```

Comment:
My intuition is telling me that you shouldn't use arrow functions inside object literals but remind me to look it up and I'll get back to you on it.